data/HOP/data.json

```diff
@@ -12,12 +12,6 @@
     },
     "optimism": {
       "address": "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc"
-    },
-    "goerli": {
-      "address": "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc"
-    },
-    "optimism-goerli": {
-      "address": "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc"
```

Comment:
This looks like a bug I wonder how this happewned?  I think we need to do a major version bump

Reply:
yeah, good call